pages/dao/[symbol]/proposal/components/TokenAccountSelect.tsx

```diff
@@ -27,34 +32,23 @@ const TokenAccountSelect = ({
     const { mint, uiAmount, mintName, isATA } = ownedTokenAccountsInfo[
       pubkeyString
     ];
+    const details = {
```

Comment:
Objects doesn't ensure the attributes order. Seems like to me that an array would be more appropriate:

```
const details = [
{ 
    label: 'Mint Name',
    text: mintName,
},
{ 
    label:  'UI Balance',
    text: uiAmount.toString(),
},
{ 
    label: 'Mint',
    text: mint.toBase58(),
},
];
```